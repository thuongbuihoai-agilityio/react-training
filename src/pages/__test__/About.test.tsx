import { render, screen } from "@testing-library/react";
import About from "../../../pages/about";

describe("About page", () => {
  test("renders a heading", () => {
    render(<About />);
    const heading = screen.getByText(/About page/i);
    expect(heading).toBeInTheDocument();
  });

  test("renders homepage unchanged", () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});
