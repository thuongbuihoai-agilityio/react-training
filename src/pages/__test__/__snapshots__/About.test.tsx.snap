// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders homepage unchanged 1`] = `
<div>
  <h1>
    About page
  </h1>
</div>
`;
